policy/cnspec_policy.proto

```diff
@@ -662,3 +662,13 @@ message PurgeAssetsConfirmation {
   map<string, string> errors = 2;
 }
 
+// **********       Public Report        **************
```

Comment:
Shared reporting belong into upstream/upstream.proto similar to https://github.com/mondoohq/cnquery/tree/main/upstream